--- gdbm-1.11/doc/gdbm.texinfo.orig	2013-12-26 18:51:05.253855816 +0100
+++ gdbm-1.11/doc/gdbm.texinfo	2013-12-26 18:51:46.497187418 +0100
@@ -6,11 +6,11 @@
 @settitle GDBM manual
 
 @ifinfo
-@dircategory Programming & development tools
+@dircategory Libraries:
 @direntry
-* GDBM: (gdbm).                 The GNU database manager.
-* gdbm_dump: gdbm_dump(gdbm).   Dump the GDBM database into a flat file.
-* gdbm_load: gdbm_load(gdbm).   Load the database from a flat file.
+* GDBM: (gdbm).				The GNU database manager
+* gdbm_dump: gdbm_dump(gdbm).		Dump the GDBM database into a flat file
+* gdbm_load: gdbm_load(gdbm).		Load the database from a flat file
 @end direntry
 @end ifinfo
 
