--- gdbm-1.13/export/export.c.orig	2017-01-02 12:51:49.000000000 +0100
+++ gdbm-1.13/export/export.c	2017-04-10 21:29:05.764751534 +0200
@@ -23,6 +23,7 @@
 
 #include <gdbm.h>
 
+#define GDBM_EXPORT_18
 /* Pull in gdbm_export() */
 #include "gdbmexp.c"
 
--- gdbm-1.13/src/gdbmexp.c.orig	2017-01-02 12:51:49.000000000 +0100
+++ gdbm-1.13/src/gdbmexp.c	2017-04-10 21:35:32.624747118 +0200
@@ -21,8 +21,13 @@
 # include "autoconf.h"
 # include <arpa/inet.h>
 
+#ifdef GDBM_EXPORT_18
+# define GDBM_SET_ERRNO(dbf, ec, fatal) gdbm_errno = ec
+# define GDBM_ASSERT_CONSISTENCY(dbf, val)
+#else
 # include "gdbmdefs.h"
 # include "gdbm.h"
+#endif
 
 int
 gdbm_export_to_file (GDBM_FILE dbf, FILE *fp)
