--- gdbm-1.8.0/Makefile.in.orig	Wed May 19 05:16:06 1999
+++ gdbm-1.8.0/Makefile.in	Thu May 20 10:30:07 1999
@@ -45,6 +45,8 @@
 man3dir = $(prefix)/man/man3
 manext = 3
 
+DESTDIR =
+
 #### End of system configuration section. ####
 
 .SUFFIXES:
@@ -120,20 +122,21 @@
 progs: $(PROGS)
 
 install: libgdbm.la gdbm.h gdbm.info
-	$(srcdir)/mkinstalldirs $(libdir) $(includedir) $(man3dir) $(infodir)
-	$(LIBTOOL) install -c libgdbm.la $(libdir)/libgdbm.la
-	$(INSTALL_DATA) -o $(BINOWN) -g $(BINGRP) gdbm.h $(includedir)/gdbm.h
-	$(INSTALL_DATA) -o $(BINOWN) -g $(BINGRP) $(srcdir)/gdbm.3 \
-		$(man3dir)/gdbm.3
-	$(INSTALL_DATA) -o $(BINOWN) -g $(BINGRP) $(srcdir)/gdbm.info \
-		$(infodir)/gdbm.info
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(libdir) $(DESTDIR)$(includedir) \
+		$(DESTDIR)$(man3dir) $(DESTDIR)$(infodir)
+	$(LIBTOOL) install -c libgdbm.la $(DESTDIR)$(libdir)/libgdbm.la
+	$(INSTALL_DATA) gdbm.h $(DESTDIR)$(includedir)/gdbm.h
+	$(INSTALL_DATA) $(srcdir)/gdbm.3 \
+		$(DESTDIR)$(man3dir)/gdbm.3
+	$(INSTALL_DATA) $(srcdir)/gdbm.info \
+		$(DESTDIR)$(infodir)/gdbm.info
 
 install-compat:
 	$(srcdir)/mkinstalldirs $(includedir)
-	$(INSTALL_DATA) -o $(BINOWN) -g $(BINGRP) $(srcdir)/dbm.h \
-		$(includedir)/dbm.h
-	$(INSTALL_DATA) -o $(BINOWN) -g $(BINGRP) $(srcdir)/ndbm.h \
-		$(includedir)/ndbm.h
+	$(INSTALL_DATA) $(srcdir)/dbm.h \
+		$(DESTDIR)$(includedir)/dbm.h
+	$(INSTALL_DATA) $(srcdir)/ndbm.h \
+		$(DESTDIR)$(includedir)/ndbm.h
 
 #libgdbm.a: $(OBJS) gdbm.h
 #	rm -f libgdbm.a
