--- gdbm-1.9.1/doc/gdbm.texinfo.orig	2011-08-11 14:07:32.000000000 +0200
+++ gdbm-1.9.1/doc/gdbm.texinfo	2011-08-14 09:30:31.906384656 +0200
@@ -6,9 +6,9 @@
 @settitle gdbm manual
 
 @ifinfo
-@dircategory Programming & development tools
+@dircategory Libraries:
 @direntry
-* GDBM: (gdbm).                 The GNU database manager.
+* GDBM: (gdbm).				The GNU database manager
 @end direntry
 @end ifinfo
 
