--- gdbm-1.12/doc/gdbm.texinfo.orig	2016-05-21 17:19:34.711227665 +0200
+++ gdbm-1.12/doc/gdbm.texinfo	2016-05-21 17:20:33.357891871 +0200
@@ -6,11 +6,11 @@
 @settitle GDBM manual
 
 @ifinfo
-@dircategory Programming & development tools
+@dircategory Libraries:
 @direntry
-* GDBM: (gdbm).                  The GNU database manager.
-* gdbm_dump: (gdbm) gdbm_dump.   Dump the GDBM database into a flat file.
-* gdbm_load: (gdbm) gdbm_load.   Load the database from a flat file.
+* GDBM: (gdbm).				The GNU database manager
+* gdbm_dump: (gdbm) gdbm_dump.		Dump the GDBM database into a flat file
+* gdbm_load: (gdbm) gdbm_load.		Load the database from a flat file
 @end direntry
 @end ifinfo
 
