--- gdbm-1.8.3/gdbm.texinfo.orig	Tue Oct 15 23:59:38 2002
+++ gdbm-1.8.3/gdbm.texinfo	Tue Dec 24 23:43:32 2002
@@ -2,9 +2,9 @@
 @comment %**start of header (This is for running Texinfo on a region.)
 @setfilename gdbm.info
 @settitle gdbm
-@dircategory Programming & development tools
+@dircategory Libraries:
 @direntry
-* GDBM: (gdbm).			The GNU database manager.
+* GDBM: (gdbm).				The GNU database manager
 @end direntry
 @c @setchapternewpage odd
 @comment %**end of header (This is for running Texinfo on a region.)
